Engine/Makefile-defs.linux

```diff
@@ -1,12 +1,18 @@
+USE_BUILT_IN_LIBSRC = 0
+USE_MIDI_PATCH = 1
```

Comment:
it seems the midi patch is now integrated into allegro as of 4.4.3

Reply:
This behaviour is unchanged from before. I guess it depends on what version of allegro is bundled with your Linux distro